documentation/airport-guides/kaus.md

```diff
@@ -0,0 +1,65 @@
+### About this facility
+
+Austin ATCT (AUS), Austin, TX, USA  
+199,632 aircraft movements in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Airport.asp))
+
+This is the Control Tower position at AUS, serving Austin-Bergstrom International Airport, the main airport for Austin, TX.
+
+### Airspace
+
+Class C airspace extends to 17,000 feet above sea level, encompassing the area around the airport.
+
+### Instrument Procedures
+
+#### Operation
+
+##### Preferential RWY
+
+- LDG 0600-1200: RWY35L and 35R
+- T/O 0600-1200: RWY17L and 17R
+- Cargo ACFT use 17R/35L for T/O and LDG
+
+##### Traffic note
+
+Traffic to/from the West is usually assigned to RWY17R/35L Traffic to/from the East is usually assigned to RWY17L/35R
+
+### Sectors Available in OpenScope
+
+```
+E - East Approach
+S - South Approach
+W - West Approach
+```
+The airspace available in Openscope is a combination of these 3 sectors.
+
+### Airports of Interest
+
+#### KAUS (Primary Airport)
+
+Austin-Bergstrom International Airport  
+197,949 annual operations
+
+Bergstrom Airport used to be Bergstrom Air Force Base, until 1991 when it was converted to it's use today. The 12,250-ft long runway 17R/35L is inherited from the air force base, with the accompanying 17L/35R at 9,000 ft on the other side of the field. Both ends of both runways are equipped with ILS, but also have RNAV procedures.

```

Comment:
`when it was converted to it's use today`
`when it was converted to a commercial airport.`